packages/ssz/src/viewDU/listComposite.ts

```diff
@@ -66,6 +66,43 @@ export class ListCompositeTreeViewDU<
     return this.type.getViewDU(newRootNode) as this;
   }
 
+  /**
+   * Returns a new ListCompositeTreeViewDU instance with the values from `index` to the end of list
+   *
+   * ```ts
+   * const nodes = getChunkNodes()
+   * return listFromChunkNodes(node.clice(index))
+   * ```
+   *
+   * Note: If index === n, returns an empty list of length 0
+   *
+   */
+  sliceFrom(index: number): this {
+    // Commit before getting rootNode to ensure all pending data is in the rootNode
+    this.commit();
+
+    // If slicing from 0, no slicing is necesary
+    if (index <= 0) {
+      return this;
+    }
+
+    let newChunksNode;
+    let newLength;
+
+    if (index >= this.nodes.length) {
```

Comment:
I think we should raise error incase user want to slice from out of bound index.

Reply:
> I think we should raise error incase user want to slice from out of bound index.

There is a use case for `index === this.nodes.length` to return an empty list. But I agree `index > this.nodes.length` should throw an error 👍